src/js/controllers/chatController.js

```diff
@@ -1,22 +1,25 @@
 'use strict';
```

Comment:
This file is getting pretty huge.
We might want to consider trying to split it up somehow.
For now it's fine, though.